sktime/classification/distance_based/_proximity_forest.py

```diff
@@ -712,6 +751,19 @@ def __init__(
         self.entropy = None
         self._random_object = None
         super().__init__()
+        if self.n_jobs != "deprecated":
```

Comment:
I think the nessage, the condition, and the handling are not correct.

The next release should not change anything for current users, and schedule removal for 0.31.0.

If `n_jobs` is passed as a non-default, it should override the `backend_params` to result in equivalent behaviour.